extensions/ipynb/package.nls.json

```diff
@@ -2,6 +2,7 @@
 	"displayName": ".ipynb Support",
 	"description": "Provides basic support for opening and reading Jupyter's .ipynb notebook files",
 	"ipynb.pasteImagesAsAttachments.enabled": "Enable/disable pasting of images into Markdown cells in ipynb notebook files. Pasted images are inserted as attachments to the cell.",
+	"ipynb.experimental.serialization": "Experimental feature to serialize the Jupyter notebook in a worker thread.",
```

Comment:
since it only works in node extension host, let's call that out explicitly.

Reply:
I mentioned this is for desktop, would that be a good description? I was unsure how to phrase it.